packages/terra-avatar/src/Avatar.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import TerraImage from 'terra-image';
+import styles from './Avatar.scss';
+
+const cx = classNames.bind(styles);
+
+const AvatarVariants = {
+  FACILITY: 'facility',
+  USER: 'user',
+};
+
+const propTypes = {
+  /**
+   * The text content that specifies the alternative text for the image.
+   */
+  alt: PropTypes.string,
+  /**
+   * The image to display.
+   */
+  image: PropTypes.string,
+  /**
+   * The initials to display.
+   */
+  initials: PropTypes.string,
+  /**
+   * The avatar variant. One of `AvatarVariants.FACILITY`, `AvatarVariants.USER`.
+   */
+  variant: PropTypes.oneOf([AvatarVariants.FACILITY, AvatarVariants.USER]),
+};
+
+const defaultProps = {
+  variant: AvatarVariants.USER,
+};
+
+export const propsErrorMsg = 'Only one of the props: [image, initials] should be supplied.';
+
+const Avatar = ({
+  alt,
+  image,
+  initials,
+  variant,
+  ...customProps
+  }) => {
+  if (image && initials) {
+    throw new Error(propsErrorMsg);
```

Comment:
I think this should be a warning, I personally don't think presentational components should hard crash intentionally or should do so only in extremely special circumstances. Does anyone else have any opinions on this?

Reply:
Yeah, we'll want to update this to throw a warning instead of an error as to not crash the app. Take a look at the code in terra-base. We log a console warning and only if the code is running in non-prod mode: https://github.com/cerner/terra-core/blob/master/packages/terra-base/src/baseStyles.js

Reply:
So a console.warn() or is there a better way?

Reply:
Updated in this [commit](https://github.com/cerner/terra-core/commit/855d5fc5c14ce5dc2a29cb07d7d5575af2320962).